iac/environments/native-development/cat_full.tf

```diff
@@ -1,12 +1,25 @@
 module "cat_full" {
   source = "../../compositions/cat-full"
 
-  aws_account_id           = var.aws_account_id
-  aws_region               = var.aws_region
-  docker_image_tags        = var.docker_image_tags
-  environment_is_ephemeral = var.environment_is_ephemeral
-  environment_name         = var.environment_name
-  resource_name_prefixes   = var.resource_name_prefixes
-  task_container_configs   = var.task_container_configs
-  vpc_cidr_block           = var.vpc_cidr_block
+  aws_account_id                   = var.aws_account_id
+  aws_region                       = var.aws_region
+  cat_api_environment              = var.cat_api_environment
+  cat_api_ingress_cidr_safelist    = var.cat_api_ingress_cidr_safelist
+  cat_api_ssm_secret_paths         = local.cat_api_ssm_secret_paths
```

Comment:
This refers to `local.cat_api_ssm_secret_paths` but `cat_api_ssm_secret_paths` is never declared in a `locals` block anywhere. It looks however like it's declared as a `variable` in this env. Am I missing something?

Reply:
Good spot. I missed pushing the `legacy-resources.tf` but have modified it slightly to hopefully be more evident how this gets used (mainly because tfvars files can't contain resource references; if we had pre-existing ARNs then it would otherwise be fine)